bioconda_utils/cli.py

```diff
@@ -537,13 +537,13 @@ def dag(recipe_folder, config, packages="*", format='gml', hide_singletons=False
      help="""Skip updating/bumping packges that are already built with
      compatible pinnings in one of the given channels in addition to those
      listed in 'config'.""")
-@arg('--bump-only-python',
```

Comment:
@dpryan79, I hope the added `noarch` checks are enough to offset the need for this. Let me know if I didn't catch some use cases for this.